base/fhir_validator.bal

```diff
@@ -50,3 +50,13 @@ public isolated function validate(json|anydata data, typedesc<anydata>? targetFH
     }
 
 }
+
+# This method will validate parsed FHIR resource.
+# Validation consist of Structure, cardinality, Value domain, Profile.
+# This is only for internal use.
+#
+# + data - parsed FHIR resource
+# + return - If the validation fails, return validation error
+isolated function validateFhirResource(anydata data) returns FHIRValidationError? {
```

Comment:
We removed this function and its usages since a custom validator is not needed anymore - https://github.com/wso2-enterprise/open-healthcare/issues/1212

Reply:
haven't removed properly from the codegen tool? @AnjanaSenanayake

Reply:
@AnjanaSenanayake please send a separate PR to codegen tool for this.

Reply:
ack